jekyll/_components/tables.md

```diff
@@ -2,17 +2,142 @@
 layout: page
 title: Tables
 order: 9
-draft: true
 ---
 
-Tabular data appears in a variety of places, the Network panel is the most obvoious example. The console is also able to output a visual table using `console.table()` e.g. `console.table(["apples", "oranges", "bananas"]);`
-
-![console table example](../images/components/tables/console-table-example.png "Console table example")
```

Comment:
We should probably delete the `console-table-example.png` file, if there aren't any more references to it...

Reply:
Image deleted